bundles/org.openhab.core.io.rest.voice/src/main/java/org/openhab/core/io/rest/voice/internal/VoiceResource.java

```diff
@@ -133,7 +137,7 @@ public Response getInterpreter(
     }
 
     @POST
-    @Path("/interpreters/{id: [a-zA-Z_0-9]+}")
+    @Path("/interpreters/{ids: [a-zA-Z_0-9,]+}")
```

Comment:
For compatibility reason, I think we should keep `id`.

Reply:
Since this is internal only, you can change to `ids` without any issue.

Reply:
Yes, correct. My comment was useless. You can ignore it.